src/aeb_fate_generate_ops.erl

```diff
@@ -163,6 +163,7 @@ ops_defs() ->
 
     , { 'ECVERIFY',            16#72,  false,    true,   true,  1300, [a,a,a,a],           ecverify, {bytes, address, bytes}, boolean, "Arg0 := ecverify(Hash, PubKey, Signature)"}
     , { 'ECVERIFY_SECP256K1',  16#73,  false,    true,   true,  1300, [a,a,a,a], ecverify_secp256k1,   {bytes, bytes, bytes}, boolean, "Arg0 := ecverify_secp256k1(Hash, PubKey, Signature)"}
+    , { 'ECRECOVER_SECP256K1', 16#7d,  false,    true,   true,  1300, [a,a,a,a], ecrecover_secp256k1,   {bytes, bytes, bytes}, bytes,  "Arg0 := ecrecover_secp256k1(Hash, V, R, S)"}
```

Comment:
Put it in op code order please (or the next person adding an instruction will choose the same op code).

Also shouldn't it only take a hash and a signature? So `[a,a,a]` and `{bytes, bytes}`.

Reply:
Given that ecverify takes a single 65 byte signature it would be very strange to have ecrecover take the parts separately.